numalogic/udfs/tools.py

```diff
@@ -270,6 +304,7 @@ def ack_read(
         Acknowledge the read message. Return True when the msg has to be trained.
         Args:
             key: key
+            pipeline_id: key
```

Comment:
Here also can we do like we discussed?

Reply:
Let me remove it from the docstring. I might have missed to remove it.